src/main/java/no/entur/uttu/graphql/model/StopPlace.java

```diff
@@ -0,0 +1,8 @@
+package no.entur.uttu.graphql.model;
+
+import org.rutebanken.netex.model.MultilingualString;
+import org.rutebanken.netex.model.Quay;
+
+import java.util.List;
+
+public record StopPlace(String id, MultilingualString name, String transportMode, String stopPlaceType, List<Quay> quays) {}
```

Comment:
Wasn't sure if this needs to be somehow specially named to avoid mixing with StopPlace classes from other packages